agent/connect/csr.go

```diff
@@ -67,6 +70,13 @@ func CreateCSR(uri CertURI, commonName string, privateKey crypto.Signer,
 	return csrBuf.String(), nil
 }
 
+// CreateCSR returns a CSR to sign the given service along with the PEM-encoded
+// private key for this certificate.
+func CreateCSR(uri CertURI, commonName string, privateKey crypto.Signer,
```

Comment:
Is there a need to keep this function? Is it still being used anywhere?

Also, if it's not being used anywhere, what if the new CreateCSRWithSAN is renamed to keep the old "CreateCSR" name?

Reply:
Good catch! Did what you suggested!